test/sharness/t0066-migration.sh

```diff
@@ -30,17 +41,25 @@ test_expect_success "output looks good" '
   grep "Please get fs-repo-migrations from https://dist.ipfs.io" false_out
 '
 
+# The migrations will succeed, but the daemon will still exit with 1 because
+# the fake migrations do not update the repo version number.
+#
+# If run with real migrations, the daemon continues running and must be killed.
 test_expect_success "ipfs daemon --migrate=true runs migration" '
-  test_expect_code 1 ipfs daemon --migrate=true > true_out
+  test_expect_code 1 ipfs daemon --migrate=true > true_out 2>&1
 '
 
-test_expect_failure "output looks good" '
-  grep "Running: " true_out > /dev/null &&
-  grep "Success: fs-repo has been migrated to version 5." true_out > /dev/null
+test_expect_success "output looks good" '
+  grep "applying 7-to-8 repo migration" true_out > /dev/null &&
+  grep "applying 8-to-9 repo migration" true_out > /dev/null &&
+  grep "applying 9-to-10 repo migration" true_out > /dev/null &&
+  grep "applying 10-to-11 repo migration" true_out > /dev/null &&
+  grep "Success: fs-repo migrated to version 11" true_out > /dev/null
 '
 
+# Daemon exits with 1 after the prompt gets "n" when asking to migrate. 
 test_expect_success "'ipfs daemon' prompts to auto migrate" '
-  test_expect_code 1 ipfs daemon > daemon_out 2> daemon_err
+  test_expect_code 1 bash -c "echo n | ipfs daemon > daemon_out 2> daemon_err"
```

Comment:
Why has this part of the test changed? Isn't the prompt for auto-migration code still the same?

Reply:
The prompt is the same, but needed to `echo n` to answer the prompt so that it would not just hang.  Needed the `bash -c ...` to handle the pipe in the command.

Reply:
After retesting, I do not know why this change seemed necessary.  Changed back to original.